### DISCUSSION TITLE
certbot issue - .well-known directory not accessible

### DISCUSSION THREAD

**Discussion by niklasenB** [langgenius/dify#7171] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

I am following the README from certbot/ 
running `sudo docker-compose exec -it certbot /bin/sh /update-cert.sh`
returns a 404 despite the instance is accessible through http 

Invalid response from {my_domain}/.well-known/acme-challenge/hreC07rK62VdfsmYbsckgoUHRjxKy_Thg5TW81qT--w: 404

how to solve this?

**Comment by niklasenB** [comment]
anyone knows how to fix this?

**Reply by Rumdaz** [reply]
> anyone knows how to fix this?
I've encountered the same issue as well. Have you resolved it yet?
